### PR TITLE
Add support for Bling65 PCB

### PR DESCRIPTION
**Pull request by Deemen17**
## Description

Add support for Bling65 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/deemen17/bling65/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┼───┤
+     * │Ctrl│GUI │Alt │                        │Alt│ Fn│Ctl│ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┴───┘
+     */
+    [0] = LAYOUT_65_ansi(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (waffle87) [inline keyboards/deemen17/bling65/keymaps/default/keymap.c:37]:
```suggestion
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    KC_RALT, _______,   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/deemen17/bling65/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬┴──┬───┼───┼───┤
+     * │Ctrl│GUI │Alt │                        │Alt│ Fn│Ctl│ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┴───┘
+     */
+    [0] = LAYOUT_65_ansi(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    KC_RALT, MO(1),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (waffle87) [inline keyboards/deemen17/bling65/keymaps/via/keymap.c:37]:
```suggestion
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                    KC_RALT, _______,   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

**Review by waffle87 — CHANGES_REQUESTED** [review]
There are also submodule changes that need to be reverted.

**Comment by drashna** [comment]
For the submodule, you should be able to cd into that directory and run `git checkout qmk-master`, cd out of it, and commit the changes.

**Comment by Deemen17** [comment]
> For the submodule, you should be able to cd into that directory and run `git checkout qmk-master`, cd out of it, and commit the changes.

Did I leave out the submodule change? I'm a bit new to git so I'm not sure yet

**Review by drashna — APPROVED** [review]

File: keyboards/deemen17/bling65/rules.mk
Comment (fauxpark) [inline keyboards/deemen17/bling65/rules.mk:1]: This file can be deleted.

File: keyboards/deemen17/bling65/info.json
Comment (fauxpark) [inline keyboards/deemen17/bling65/info.json:1]: This file should be renamed to `keyboard.json`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by tzarc** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

**Pull request closed without merging**